### PR TITLE
Update rkhunter from 1.4.4 to 1.4.4-1

### PR DESCRIPTION
**Pull request by uberhacker**

### CODE REVIEW

File: packages/rkhunter.rb [hunk 1]
```diff
@@ -3,32 +3,30 @@
...
+    system "sed -i 's,/var\",#{CREW_PREFIX}/share\",g' installer.sh"
+    system "sed -i 's,/var/,#{CREW_PREFIX}/share/,g' installer.sh"
+    system "bash installer.sh --layout #{CREW_PREFIX} --install"
+    system "mkdir -p #{CREW_DEST_PREFIX}/bin"
+    system "mkdir -p #{CREW_DEST_PREFIX}/etc"
+    system "mkdir -p #{CREW_DEST_LIB_PREFIX}"
+    system "mkdir -p #{CREW_DEST_PREFIX}/share/doc"
+    system "mkdir -p #{CREW_DEST_PREFIX}/share/lib"
+    system "mkdir -p #{CREW_DEST_PREFIX}/share/man/man8"
+    system "cp #{CREW_PREFIX}/bin/rkhunter #{CREW_DEST_PREFIX}/bin"
```
Comment (cstrouse) [inline packages/rkhunter.rb]: Can you consolidate the mkdir and the cp into a single command for the binary, for the config file, and for the manpage using the install cmd?

**Comment by cstrouse** [comment]
Works on Samsung Chromebook Plus (aarch64).

File: packages/rkhunter.rb [hunk 2]
```diff
@@ -3,32 +3,28 @@
...
-    system "cp /usr/local/bin/rkhunter #{CREW_DEST_DIR}/usr/local/bin"
-    system "cp /usr/local/etc/rkhunter.conf #{CREW_DEST_DIR}/usr/local/etc"
-    system "cp -r /usr/local/lib64/rkhunter #{CREW_DEST_DIR}/usr/local/lib64"
-    system "cp -r /usr/local/share/doc/rkhunter-1.4.4 #{CREW_DEST_DIR}/usr/local/share/doc"
-    system "cp -r /usr/local/share/lib/rkhunter #{CREW_DEST_DIR}/usr/local/share/lib"
-    system "cp /usr/local/share/man/man8/rkhunter.8* #{CREW_DEST_DIR}/usr/local/share/man/man8"
+    system "sed -i 's,/var\",#{CREW_PREFIX}/share\",g' installer.sh"
+    system "sed -i 's,/var/,#{CREW_PREFIX}/share/,g' installer.sh"
+    system "bash installer.sh --layout #{CREW_PREFIX} --install"
+    system "gzip -9 #{CREW_PREFIX}/share/man/man8/rkhunter.8"
```
Comment (cstrouse) [inline packages/rkhunter.rb]: You can't just gzip manpages according to the documentation at BLFS for compressdoc.

**Comment by cstrouse** [comment]
This updated version fails to install with the following errors:

```bash
Checking system for:
 Rootkit Hunter installer files: found
chown: changing ownership of './files': Operation not permitted
chown: changing ownership of './files/rkhunter': Operation not permitted
chown: changing ownership of './files/i18n': Operation not permitted
chown: changing ownership of './files/i18n/en': Operation not permitted
chown: changing ownership of './files/i18n/zh': Operation not permitted
chown: changing ownership of './files/i18n/ja': Operation not permitted
chown: changing ownership of './files/i18n/cn': Operation not permitted
chown: changing ownership of './files/i18n/de': Operation not permitted
chown: changing ownership of './files/i18n/zh.utf8': Operation not permitted
chown: changing ownership of './files/i18n/tr': Operation not permitted
chown: changing ownership of './files/i18n/tr.utf8': Operation not permitted
chown: changing ownership of './files/programs_bad.dat': Operation not permitted
chown: changing ownership of './files/CHANGELOG': Operation not permitted
chown: changing ownership of './files/README': Operation not permitted
chown: changing ownership of './files/ACKNOWLEDGMENTS': Operation not permitted
chown: changing ownership of './files/rkhunter.spec': Operation not permitted
chown: changing ownership of './files/LICENSE': Operation not permitted
chown: changing ownership of './files/mirrors.dat': Operation not permitted
chown: changing ownership of './files/stat.pl': Operation not permitted
chown: changing ownership of './files/backdoorports.dat': Operation not permitted
chown: changing ownership of './files/filehashsha.pl': Operation not permitted
chown: changing ownership of './files/signatures': Operation not permitted
chown: changing ownership of './files/signatures/RKH_BillGates.ldb': Operation not permitted
chown: changing ownership of './files/signatures/RKH_sshd.ldb': Operation not permitted
chown: changing ownership of './files/signatures/RKH_libncom.ldb': Operation not permitted
chown: changing ownership of './files/signatures/RKH_dso.ldb': Operation not permitted
chown: changing ownership of './files/signatures/RKH_kbeast.ldb': Operation not permitted
chown: changing ownership of './files/signatures/RKH_jynx.ldb': Operation not permitted
chown: changing ownership of './files/signatures/RKH_iptablex.ldb': Operation not permitted
chown: changing ownership of './files/signatures/RKH_xsyslog.ldb': Operation not permitted
chown: changing ownership of './files/signatures/RKH_libkeyutils1.ldb': Operation not permitted
chown: changing ownership of './files/signatures/RKH_Glubteba.ldb': Operation not permitted
chown: changing ownership of './files/signatures/RKH_sniffer.ldb': Operation not permitted
chown: changing ownership of './files/signatures/RKH_pamunixtrojan.ldb': Operation not permitted
chown: changing ownership of './files/signatures/RKH_MMD-0028-2014.ldb': Operation not permitted
chown: changing ownership of './files/signatures/RKH_libkeyutils.ldb': Operation not permitted
chown: changing ownership of './files/signatures/RKH_shv.ldb': Operation not permitted
chown: changing ownership of './files/signatures/RKH_turtle.ldb': Operation not permitted
chown: changing ownership of './files/rkhunter.8': Operation not permitted
chown: changing ownership of './files/check_modules.pl': Operation not permitted
chown: changing ownership of './files/FAQ': Operation not permitted
chown: changing ownership of './files/rkhunter.conf': Operation not permitted
chown: changing ownership of './files/contrib': Operation not permitted
chown: changing ownership of './files/contrib/run_rkhunter.sh': Operation not permitted
chown: changing ownership of './files/contrib/README.txt': Operation not permitted
chown: changing ownership of './files/contrib/rkhunter_remote_howto.txt': Operation not permitted
chown: changing ownership of './files/readlink.sh': Operation not permitted
chown: changing ownership of './files/suspscan.dat': Operation not permitted
 A web file download command: wget found
Starting installation:
 Checking installation directory "/usr/local": it exists and is writable.
 Checking installation directories:
  Directory /usr/local/share/doc/rkhunter-1.4.4: creating: OK
  Directory /usr/local/share/man/man8: exists and is writable.
  Directory /usr/local/etc: exists and is writable.
  Directory /usr/local/bin: exists and is writable.
  Directory /usr/local/lib: exists and is writable.
  Directory /usr/local/share/lib: exists and is writable.
  Directory /usr/local/lib/rkhunter/scripts: creating: OK
  Directory /usr/local/share/lib/rkhunter/db: exists and is writable.
  Directory /usr/local/share/lib/rkhunter/tmp: creating: OK
  Directory /usr/local/share/lib/rkhunter/db/i18n: creating: OK
  Directory /usr/local/share/lib/rkhunter/db/signatures: creating: OK
 Installing check_modules.pl: OK
 Installing filehashsha.pl: OK
 Installing stat.pl: OK
 Installing readlink.sh: OK
 Installing backdoorports.dat: OK
 Installing mirrors.dat: OK
 Installing programs_bad.dat: OK
 Installing suspscan.dat: OK
 Installing rkhunter.8: OK
 Installing ACKNOWLEDGMENTS: OK
 Installing CHANGELOG: OK
 Installing FAQ: OK
 Installing LICENSE: OK
 Installing README: OK
 Installing language support files: OK
 Installing ClamAV signatures: OK
 Installing rkhunter: OK
 Installing rkhunter.conf: OK
Installation complete
cp: cannot open '/usr/local/share/lib/rkhunter/db/rkhunter_prop_list.dat' for reading: Permission denied
rkhunter failed to install: `cp -r /usr/local/share/lib/rkhunter /usr/local/tmp/crew/dest/usr/local/share/lib` exited with 1
```

File: packages/rkhunter.rb (hunk 2 above)
Comment (uberhacker) [reply packages/rkhunter.rb]: I could but in this case I only need the one file compressed.  The compressdoc command does entire directories and I thought it was a bit of an overkill.  It also removes the dependency requirement.

**Comment by uberhacker** [comment]
Is this aarch64 (aka armv8)?  I tested on armv7l, i686 and x86_64 and they all worked fine.

**Comment by cstrouse** [comment]
Yes this is on aarch64. It worked properly before the modifications I asked for and now does not.

**Comment by uberhacker** [comment]
That will teach you to request a fix on what isn't broken. ;)

**Comment by cstrouse** [comment]
I just thought we should start cleaning up the places where we have a two step mkdir/cp operation by using the install command.

**Comment by uberhacker** [comment]
There must be something wrong with the latest changes then.

**Comment by uberhacker** [comment]
I don't see anything wrong with the latest updates.  Did you want to just restore to the previous version rather than wrestling with it?

**Comment by cstrouse** [comment]
Yeah let's just do that.

**Pull request merged**